specs/phase0/fork-choice.md

```diff
@@ -142,47 +175,58 @@ def get_ancestor(store: Store, root: Root, slot: Slot) -> Root:
     block = store.blocks[root]
     if block.slot > slot:
         return get_ancestor(store, block.parent_root, slot)
-    elif block.slot == slot:
-        return root
     else:
-        # root is older than queried slot, thus a skip slot. Return most recent root prior to slot
+        # Return highest block's root with slot less than or equal to the queried slot
         return root
 ```
 
+#### `get_ancestor_node`
+
+```python
+def get_ancestor_node(store: Store, node_key: Root, slot: Slot) -> Root:
+    node = store.block_slot_tree[node_key]
+    if node.slot > slot:
+        return get_ancestor_node(store, node.parent_node, slot)
+    else:
+        # Return highest node's key with slot less than or equal to the queried slot
```

Comment:
nitpick, but it seems like this comment applies to the whole function `get_ancestor_node` and might be clearer at function body level